### DISCUSSION TITLE
Dynamic Axis Scaling in Animated Charts

### DISCUSSION THREAD

**Discussion by philippulus** [chartist-js/chartist#1371] [Ideas]
Hi community,

I am looking for creating animated charts with dynamically scaling axis, changing X and Y values and moving symbols like in [this example](https://www.linkedin.com/posts/jameseagle_gold-finance-economics-activity-6987774486183845888-gtoy). Is that something apexchart can do?

https://user-images.githubusercontent.com/89039695/198819714-51932266-392a-417d-b151-72b505d38373.mp4

**Comment by gionkunz** [comment]
Hi there! :-) This is the discussions page of Chartist and not apexchart and if you are looking for animations like that apexchart might be the better choice, yes ;-)
